easel/app.js

```diff
@@ -74,10 +78,13 @@ function sortPoints(points) {
   }
 
   if (points.length) {
-    // close the path if first and last point are close enough to make the path open due to dxf -> svg floating point inaccuracies
-    if (Math.abs(points[0].x - points[points.length - 1].x) < diff && Math.abs(points[0].y - points[points.length - 1].y) < diff ) {
-      points[points.length - 1] = points[0];
-    }
+    points = points.map(subPoints => {
+      // close the path if first and last point are close enough to make the path open due to dxf -> svg floating point inaccuracies
+      if (Math.abs(subPoints[0].x - subPoints[subPoints.length - 1].x) < diff && Math.abs(subPoints[0].y - subPoints[subPoints.length - 1].y) < diff ) {
```

Comment:
Did the structure of points change? I'm not seeing any changes to it from the diff but from the old code it looks like points is an array of subpoints `[{ x, y }]` but from the changes it's being accessed like an array of subpoint arrays `[[{x, y }, {x, y}], [{x, y}]]`

Reply:
The old code in https://github.com/inventables/dxfimport/pull/7 was wrong, and the old changes did not fix the issue.
If you look at https://github.com/inventables/dxfimport/blob/e118e3686d1cab89787eede7302f07f4194641a4/easel/app.js#L50 you can see the structure has always been an array of subpoints. Do not use https://github.com/inventables/dxfimport/pull/7 as a reference please.